Add new cmdlet files (#9794)
MicrosoftDocs/office-docs-powershell: teams/teams-ps/teams/Get-CsBatchTeamsDeployment.md

---
external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
Module Name: MicrosoftTeams
online version: https://docs.microsoft.com/powershell/module/teams/get-csbatchteamsdeployment
schema: 2.0.0
---

# Get-CsBatchTeamsDeployment

## SYNOPSIS

This cmdlet is used to get the status of the batch deployment orchestration.

## SYNTAX

```
Get-CsBatchTeamsDeployment -OrchestrationId <String> [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION
After deploying teams using New-CsBatchTeamsDeployment, an admin can check the status of the job/orchestration using Get-CsBatchTeamsDeployment. 

> To learn more, see Deploy Teams at scale for frontline workers (https://docs.microsoft.com/en-us/microsoft-365/frontline/deploy-teams-at-scale?view=o365-worldwide).

Reviewer: ```suggestion
To learn more, see Deploy Teams at scale for frontline workers (https://docs.microsoft.com/microsoft-365/frontline/deploy-teams-at-scale).

```

---
external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
Module Name: MicrosoftTeams
online version: https://docs.microsoft.com/powershell/module/teams/get-csbatchteamsdeployment
schema: 2.0.0
---

# Get-CsBatchTeamsDeployment

## SYNOPSIS

This cmdlet is used to get the status of the batch deployment orchestration.

## SYNTAX

```
Get-CsBatchTeamsDeployment -OrchestrationId <String> [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION
After deploying teams using New-CsBatchTeamsDeployment, an admin can check the status of the job/orchestration using Get-CsBatchTeamsDeployment. 

> To learn more, see Deploy Teams at scale for frontline workers (https://docs.microsoft.com/microsoft-365/frontline/deploy-teams-at-scale).

Reviewer 2: I would use markdown for this, i.e.:

```
see [Deploy Teams at scale for frontline workers](https://docs.microsoft.com.......)].

Author: Updated. Thanks.

## EXAMPLES

### EXAMPLE 1
```
Get-CsBatchTeamsDeployment -OrchestrationId "My-Orchestration-Id"
```

> 

Reviewer 2: Can you add a brief description of the example?

---
external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
Module Name: MicrosoftTeams
online version: https://docs.microsoft.com/powershell/module/teams/get-csbatchteamsdeployment
schema: 2.0.0
---

# Get-CsBatchTeamsDeployment

## SYNOPSIS

This cmdlet is used to get the status of the batch deployment orchestration.

## SYNTAX

```
Get-CsBatchTeamsDeployment -OrchestrationId <String> [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION
After deploying teams using New-CsBatchTeamsDeployment, an admin can check the status of the job/orchestration using Get-CsBatchTeamsDeployment. 

To learn more, see [Deploy Teams at scale for frontline workers](https://docs.microsoft.com/microsoft-365/frontline/deploy-teams-at-scale).

> 
> 

Reviewer 3: ```suggestion
```

## EXAMPLES

### EXAMPLE 1
```
Get-CsBatchTeamsDeployment -OrchestrationId "My-Orchestration-Id"
```

> This command provides the status of specified batch deployment orchestrationId.

Reviewer 3: ```suggestion
This command provides the status of the specified batch deployment orchestration ID.

```

## PARAMETERS

### OrchestrationId

> This ID is generated when a batch deployment is submitted with New-CsBatchTeamsDeployment cmdlet.

Reviewer 3: ```suggestion
This ID is generated when a batch deployment is submitted with the New-CsBatchTeamsDeployment cmdlet.

```

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

## OUTPUTS

> ###Status of the orchestrationId

Reviewer 3: ```suggestion
### Status of the orchestration ID

```

Running: The orchestration is running.

> Completed: The orchestration is completed, either succeeded, partially succeeded or Failed.

Reviewer 3: ```suggestion
Completed: The orchestration is completed, either succeeded, partially succeeded, or failed.

```

> 
> 

Reviewer 3: ```suggestion
```
